### ISSUE TITLE
Release file not gziped

### ISSUE THREAD

**Issue by massada** [docker/docker-credential-helpers#123]
The pass release is not gziped.

```shell
$ tar zxvf docker-credential-pass-v0.6.0-amd64.tar.gz 
gzip: stdin: not in gzip format
tar: Child returned status 1
tar: Error is not recoverable: exiting now
```

Untaring without the z option works.

```shell
$ tar xvf docker-credential-pass-v0.6.0-amd64.tar.gz 
docker-credential-pass
$ ./docker-credential-pass
Usage: ./docker-credential-pass <store|get|erase|list|version>
```

**Comment by benvdh** [comment]
same goes for the DBUS secretsservice one..

**Comment by newey01c** [comment]
This is still the case.

**Comment by thaJeztah** [comment]
Looks to be fixed for v0.6.2 and up;

```bash
$ curl -L -O https://github.com/docker/docker-credential-helpers/releases/download/v0.6.2/docker-credential-pass-v0.6.2-amd64.tar.gz
$ tar zxvf docker-credential-pass-v0.6.2-amd64.tar.gz
$ chmod +x docker-credential-pass
$ ./docker-credential-pass version
0.6.2
```
